src/devices/tuya.ts

```diff
@@ -5337,7 +5337,7 @@ const definitions: Definition[] = [
         ],
     },
     {
-        fingerprint: [{modelID: 'TS0601', manufacturerName: '_TZE200_bcusnqt8'}],
+        fingerprint: tuya.fingerprint('TS0601', ['_TZE200_bcusnqt8']),
```

Comment:
Since this device has only 1 model/fingerprint, I propose to update the `vendor` and `model` instead (and remove the whitelabel)

Reply:
Make sens. I don't know if this device is sold by another brand. Should I push a new PR with the change or you update directly on the main ? Thanks

Reply:
just update this directly.

Reply:
Done, sorry I didn't know I can update the PR. Thanks